client/src/components/generic/Header.js

```diff
@@ -76,8 +76,6 @@ export const Header = ({ hideEmployers = false }) => {
   const classes = useStyles();
   const [keycloak] = useKeycloak();
 
-  const handleLogoClick = () => history.push(Routes.EmployerForm);
```

Comment:
What is the intent here?

Reply:
This was the hacky way of handling the logo before, we added a link around the image so this is unnecessary and impossible to actually activate

Reply:
Also it was a failure for the jsx-a11y ruleset